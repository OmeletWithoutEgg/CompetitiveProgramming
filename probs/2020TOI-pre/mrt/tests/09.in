71
43 12 15 11 4 32 10
3 2 1 2 1 1 2
17 16 3 9 14 2 14
3 2 1 0 1 1 1
19 8 3 6 18 17 6
2 1 1 1 1 0 1
41 12 24 14 5 39 22
59 24 32 58 19 21 48
23 4 19 20 7 15 21
31 9 30 7 8 25 15
11 1 4 0 2 3 10
53 51 40 29 37 5 46
37 19 28 1 23 3 36
43 6 17 5 0 35 39
67 57 26 35 26 19 53
47 43 12 30 14 3 36
29 17 22 17 27 27 3
13 8 0 0 0 10 12
7 5 1 3 6 5 2
11 5 8 1 6 4 9
13 11 1 9 7 11 7
43 34 35 23 27 19 31
17 4 10 12 1 1 7
3 1 2 2 0 1 1
47 23 13 45 37 32 10
31 16 21 25 22 26 23
67 42 2 66 19 28 45
47 39 41 10 16 19 11
29 13 28 9 27 2 27
59 40 35 51 41 37 22
29 4 15 16 12 6 9
43 22 10 12 14 11 11
43 11 31 4 28 0 17
2 0 1 1 0 0 1
37 31 8 21 2 35 7
23 4 13 19 18 17 16
19 6 16 16 3 2 13
53 40 23 46 6 3 8
43 5 0 1 7 19 17
17 7 5 14 5 6 12
47 39 14 19 0 24 43
31 15 6 7 12 22 14
53 0 35 19 2 32 7
2 0 1 0 1 1 0
59 22 54 4 41 19 30
13 4 4 10 2 11 12
29 10 6 15 15 3 18
17 7 6 6 3 9 13
2 0 1 0 1 1 0
7 3 4 5 5 1 6
17 16 9 6 10 9 10
67 49 31 49 34 33 47
47 15 9 22 41 32 13
31 19 27 6 27 3 7
23 1 4 14 18 0 2
5 1 4 0 2 1 4
5 0 2 3 2 4 4
37 22 26 16 16 13 17
23 1 14 21 2 12 11
3 2 1 2 2 0 2
37 22 25 9 8 27 10
53 43 45 11 43 51 51
29 3 22 11 9 13 5
3 2 1 0 2 2 2
13 0 1 2 10 12 0
37 28 18 4 34 19 3
53 19 5 13 8 34 29
17 5 7 6 10 8 10
47 13 15 33 12 2 40
43 29 11 32 34 42 14
2 1 1 0 1 1 1
