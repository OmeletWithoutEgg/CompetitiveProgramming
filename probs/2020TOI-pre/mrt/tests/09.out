15 42
1 2
6 14
2 1
3 4
1 1
21 36
55 7
17 20
18 14
9 4
15 29
29 20
12 22
36 62
42 46
7 22
6 2
3 6
7 1
6 7
2 5
5 11
1 1
38 8
12 9
13 20
45 7
18 27
49 13
1 22
29 22
17 10
0 1
27 20
11 10
5 0
21 1
18 26
6 12
46 16
18 30
31 27
0 1
25 2
1 3
2 11
15 16
0 1
5 5
2 10
10 28
14 7
1 4
15 6
3 3
2 0
4 20
14 9
1 2
17 13
8 0
9 13
1 2
0 12
26 28
7 2
3 10
18 21
16 23
1 1
